### ISSUE TITLE
create_qp segfault 

### ISSUE THREAD

**Issue by hiroyuki-sato** [hiroyuki-sato/rb_rdma#5]
```
ruby -I lib private/test.rb 
pib_1
pib_0
open device pib_0
context addr 0x7f97aa425d80
ibv_ctx in pd 0x7f97aa425d80
alloc_pd 0x7f97aa425ef0
pd in mr 0x7f97aa425ef0
#<RbRDMA::MR:0x007f97aa386078>
-877658112
-880803840
context in cq 0x7f97aa425d80
alloc cq 0x7f97aa4b0b00
#<RbRDMA::CQ:0x007f97aa384ac0>
ibv_qp_init_attr 0x7f97aa2eb6a0
***************
pd 0x7f97aa425ef0
send_cq 0x7f97aa4b0b00
recv_cq 0x7f97aa4b0b00
test
private/test.rb:18: [BUG] Segmentation fault at 0x00000000000000
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-linux]

-- Control frame information -----------------------------------------------
c:0004 p:---- s:0020 e:000019 CFUNC  :initialize
c:0003 p:---- s:0018 e:000017 CFUNC  :new
c:0002 p:0200 s:0010 E:000810 EVAL   private/test.rb:18 [FINISH]
c:0001 p:0000 s:0002 E:000f00 (none) [FINISH]

-- Ruby level backtrace information ----------------------------------------
private/test.rb:18:in `<main>'
private/test.rb:18:in `new'
private/test.rb:18:in `initialize'

-- Machine register context ------------------------------------------------
 RIP: 0x00007f97a0df8b3c RBP: 0x00007ffd897ed340 RSP: 0x00007ffd897ed290
 RAX: 0x7463656a626f0079 RBX: 0x00007f97aa2eb6a0 RCX: 0x00007ffd897ed360
 RDX: 0x00007f97aa2eb6a0 RDI: 0x00007f97aa425ef0 RSI: 0x00007f97aa2eb4d0
  R8: 0x0000000000000040  R9: 0x00007ffd897ed340 R10: 0x0000000000000000
 R11: 0x00007f97a7fd020a R12: 0x0000000000000040 R13: 0x0000000000000020
 R14: 0x00007f97aa425ef0 R15: 0x00007f97aa2eb4d0 EFL: 0x0000000000010202

-- C level backtrace information -------------------------------------------
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(rb_vm_bugreport+0x51f) [0x7f97a909de0f] vm_dump.c:688
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(rb_bug_context+0xd0) [0x7f97a9078540] error.c:435
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(sigsegv+0x3e) [0x7f97a8f7371e] signal.c:890
/lib64/libpthread.so.0 [0x7f97a8a54100]
/lib64/libibverbs.so.1(ibv_cmd_create_qp+0x8c) [0x7f97a0df8b3c] src/cmd.c:718
/lib64/libpib-rdmav2.so(pib_create_qp+0x61) [0x7f97a057ca87] src/pib.c:287
/lib64/libibverbs.so.1(ibv_create_qp@@IBVERBS_1.1+0x13) [0x7f97a0dfdfa3] src/verbs.c:454
/home/vagrant/rb_rdma/lib/rb_rdma/rb_rdma.so(rdma_qp_initialize+0x14b) [0x7f97a100761b] ../../../../ext/rb_rdma/qp.c:155
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(vm_call0_body.constprop.116+0x1de) [0x7f97a9005c8e] vm_eval.c:131
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(rb_call0+0x1be) [0x7f97a90067ae] vm_eval.c:61
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(rb_class_new_instance+0x21) [0x7f97a8f165a1] object.c:1856
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(vm_call_cfunc+0xf6) [0x7f97a8ff5806] vm_insnhelper.c:1638
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(vm_call_method+0xe3) [0x7f97a90035d3] vm_insnhelper.c:2172
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(vm_exec_core+0x1fe9) [0x7f97a8ffe029] insns.def:995
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(vm_exec+0x81) [0x7f97a9001c61] vm.c:1650
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(ruby_exec_internal+0xc4) [0x7f97a8ea9c64] eval.c:245
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(ruby_run_node+0x2d) [0x7f97a8ead82d] eval.c:310
/home/vagrant/.rbenv/versions/2.3.1/bin/ruby(main+0x4b) [0x7f97a8ea98cb] addr2line.c:179

-- Other runtime information -----------------------------------------------

* Loaded script: private/test.rb

* Loaded features:

    0 enumerator.so
    1 thread.rb
    2 rational.so
    3 complex.so
    4 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/enc/encdb.so
    5 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/enc/trans/transdb.so
    6 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/unicode_normalize.rb
    7 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/rbconfig.rb
    8 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/compatibility.rb
    9 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/defaults.rb
   10 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/deprecate.rb
   11 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/errors.rb
   12 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/version.rb
   13 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/requirement.rb
   14 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/platform.rb
   15 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/basic_specification.rb
   16 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/stub_specification.rb
   17 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/util/list.rb
   18 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/stringio.so
   19 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/specification.rb
   20 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/exceptions.rb
   21 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_gem.rb
   22 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/monitor.rb
   23 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb
   24 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems.rb
   25 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/path_support.rb
   26 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/site_ruby/2.3.0/rubygems/dependency.rb
   27 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean/version.rb
   28 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean/core_ext/name_error.rb
   29 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean/levenshtein.rb
   30 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean/jaro_winkler.rb
   31 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean/spell_checkable.rb
   32 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/delegate.rb
   33 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean/spell_checkers/name_error_checkers/class_name_checker.rb
   34 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean/spell_checkers/name_error_checkers/variable_name_checker.rb
   35 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean/spell_checkers/name_error_checkers.rb
   36 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean/spell_checkers/method_name_checker.rb
   37 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean/spell_checkers/null_checker.rb
   38 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean/formatter.rb
   39 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib/did_you_mean.rb
   40 /home/vagrant/rb_rdma/lib/rb_rdma/version.rb
   41 /home/vagrant/rb_rdma/lib/rb_rdma/rb_rdma.so
   42 /home/vagrant/rb_rdma/lib/rb_rdma.rb
   43 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/prettyprint.rb
   44 /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/pp.rb

* Process memory map:

7f979b5c2000-7f979b7c5000 r--s 00000000 08:01 656766                     /usr/lib64/libc-2.17.so
7f979b7c5000-7f979b81e000 r--s 00000000 08:01 1977474                    /home/vagrant/rb_rdma/lib/rb_rdma/rb_rdma.so
7f979b81e000-7f979b859000 r--s 00000000 08:01 672211                     /usr/lib/debug/usr/lib64/libibverbs.so.1.0.0.debug
7f979b859000-7f979c6fc000 r--s 00000000 08:01 1976767                    /home/vagrant/.rbenv/versions/2.3.1/bin/ruby
7f979c6fc000-7f979c711000 r-xp 00000000 08:01 655379                     /usr/lib64/libgcc_s-4.8.5-20150702.so.1
7f979c711000-7f979c910000 ---p 00015000 08:01 655379                     /usr/lib64/libgcc_s-4.8.5-20150702.so.1
7f979c910000-7f979c911000 r--p 00014000 08:01 655379                     /usr/lib64/libgcc_s-4.8.5-20150702.so.1
7f979c911000-7f979c912000 rw-p 00015000 08:01 655379                     /usr/lib64/libgcc_s-4.8.5-20150702.so.1
7f979c912000-7f979f94c000 rw-p 00000000 00:00 0 
7f979f94c000-7f979f950000 r-xp 00000000 08:01 668401                     /usr/lib64/libipathverbs-rdmav2.so
7f979f950000-7f979fb4f000 ---p 00004000 08:01 668401                     /usr/lib64/libipathverbs-rdmav2.so
7f979fb4f000-7f979fb50000 r--p 00003000 08:01 668401                     /usr/lib64/libipathverbs-rdmav2.so
7f979fb50000-7f979fb51000 rw-p 00004000 08:01 668401                     /usr/lib64/libipathverbs-rdmav2.so
7f979fb51000-7f979fb56000 r-xp 00000000 08:01 668404                     /usr/lib64/libnes-rdmav2.so
7f979fb56000-7f979fd55000 ---p 00005000 08:01 668404                     /usr/lib64/libnes-rdmav2.so
7f979fd55000-7f979fd56000 r--p 00004000 08:01 668404                     /usr/lib64/libnes-rdmav2.so
7f979fd56000-7f979fd57000 rw-p 00005000 08:01 668404                     /usr/lib64/libnes-rdmav2.so
7f979fd57000-7f979fd5f000 r-xp 00000000 08:01 668409                     /usr/lib64/libmthca-rdmav2.so
7f979fd5f000-7f979ff5e000 ---p 00008000 08:01 668409                     /usr/lib64/libmthca-rdmav2.so
7f979ff5e000-7f979ff5f000 r--p 00007000 08:01 668409                     /usr/lib64/libmthca-rdmav2.so
7f979ff5f000-7f979ff60000 rw-p 00008000 08:01 668409                     /usr/lib64/libmthca-rdmav2.so
7f979ff60000-7f979ff67000 r-xp 00000000 08:01 668410                     /usr/lib64/libmlx4-rdmav2.so
7f979ff67000-7f97a0167000 ---p 00007000 08:01 668410                     /usr/lib64/libmlx4-rdmav2.so
7f97a0167000-7f97a0168000 r--p 00007000 08:01 668410                     /usr/lib64/libmlx4-rdmav2.so
7f97a0168000-7f97a0169000 rw-p 00008000 08:01 668410                     /usr/lib64/libmlx4-rdmav2.so
7f97a0169000-7f97a016e000 r-xp 00000000 08:01 668395                     /usr/lib64/libcxgb3-rdmav2.so
7f97a016e000-7f97a036d000 ---p 00005000 08:01 668395                     /usr/lib64/libcxgb3-rdmav2.so
7f97a036d000-7f97a036e000 r--p 00004000 08:01 668395                     /usr/lib64/libcxgb3-rdmav2.so
7f97a036e000-7f97a036f000 rw-p 00005000 08:01 668395                     /usr/lib64/libcxgb3-rdmav2.so
7f97a036f000-7f97a037a000 r-xp 00000000 08:01 668406                     /usr/lib64/libmlx5-rdmav2.so
7f97a037a000-7f97a0579000 ---p 0000b000 08:01 668406                     /usr/lib64/libmlx5-rdmav2.so
7f97a0579000-7f97a057a000 r--p 0000a000 08:01 668406                     /usr/lib64/libmlx5-rdmav2.so
7f97a057a000-7f97a057b000 rw-p 0000b000 08:01 668406                     /usr/lib64/libmlx5-rdmav2.so
7f97a057b000-7f97a057e000 r-xp 00000000 08:01 668540                     /usr/lib64/libpib-rdmav2.so
7f97a057e000-7f97a077d000 ---p 00003000 08:01 668540                     /usr/lib64/libpib-rdmav2.so
7f97a077d000-7f97a077e000 r--p 00002000 08:01 668540                     /usr/lib64/libpib-rdmav2.so
7f97a077e000-7f97a077f000 rw-p 00003000 08:01 668540                     /usr/lib64/libpib-rdmav2.so
7f97a077f000-7f97a0786000 r-xp 00000000 08:01 667760                     /usr/lib64/libcxgb4-rdmav2.so
7f97a0786000-7f97a0985000 ---p 00007000 08:01 667760                     /usr/lib64/libcxgb4-rdmav2.so
7f97a0985000-7f97a0986000 r--p 00006000 08:01 667760                     /usr/lib64/libcxgb4-rdmav2.so
7f97a0986000-7f97a0987000 rw-p 00007000 08:01 667760                     /usr/lib64/libcxgb4-rdmav2.so
7f97a0987000-7f97a09a1000 r-xp 00000000 08:01 657266                     /usr/lib64/libnl-3.so.200.16.1
7f97a09a1000-7f97a0ba0000 ---p 0001a000 08:01 657266                     /usr/lib64/libnl-3.so.200.16.1
7f97a0ba0000-7f97a0ba2000 r--p 00019000 08:01 657266                     /usr/lib64/libnl-3.so.200.16.1
7f97a0ba2000-7f97a0ba3000 rw-p 0001b000 08:01 657266                     /usr/lib64/libnl-3.so.200.16.1
7f97a0ba3000-7f97a0bea000 r-xp 00000000 08:01 657272                     /usr/lib64/libnl-route-3.so.200.16.1
7f97a0bea000-7f97a0de9000 ---p 00047000 08:01 657272                     /usr/lib64/libnl-route-3.so.200.16.1
7f97a0de9000-7f97a0dec000 r--p 00046000 08:01 657272                     /usr/lib64/libnl-route-3.so.200.16.1
7f97a0dec000-7f97a0df0000 rw-p 00049000 08:01 657272                     /usr/lib64/libnl-route-3.so.200.16.1
7f97a0df0000-7f97a0df2000 rw-p 00000000 00:00 0 
7f97a0df2000-7f97a0e03000 r-xp 00000000 08:01 667153                     /usr/lib64/libibverbs.so.1.0.0
7f97a0e03000-7f97a1002000 ---p 00011000 08:01 667153                     /usr/lib64/libibverbs.so.1.0.0
7f97a1002000-7f97a1003000 r--p 00010000 08:01 667153                     /usr/lib64/libibverbs.so.1.0.0
7f97a1003000-7f97a1004000 rw-p 00011000 08:01 667153                     /usr/lib64/libibverbs.so.1.0.0
7f97a1004000-7f97a100c000 r-xp 00000000 08:01 1977474                    /home/vagrant/rb_rdma/lib/rb_rdma/rb_rdma.so
7f97a100c000-7f97a120b000 ---p 00008000 08:01 1977474                    /home/vagrant/rb_rdma/lib/rb_rdma/rb_rdma.so
7f97a120b000-7f97a120c000 r--p 00007000 08:01 1977474                    /home/vagrant/rb_rdma/lib/rb_rdma/rb_rdma.so
7f97a120c000-7f97a120d000 rw-p 00008000 08:01 1977474                    /home/vagrant/rb_rdma/lib/rb_rdma/rb_rdma.so
7f97a120d000-7f97a1214000 r-xp 00000000 08:01 1977308                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/stringio.so
7f97a1214000-7f97a1414000 ---p 00007000 08:01 1977308                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/stringio.so
7f97a1414000-7f97a1415000 r--p 00007000 08:01 1977308                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/stringio.so
7f97a1415000-7f97a1416000 rw-p 00008000 08:01 1977308                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/stringio.so
7f97a1416000-7f97a1419000 r-xp 00000000 08:01 2105270                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/enc/trans/transdb.so
7f97a1419000-7f97a1618000 ---p 00003000 08:01 2105270                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/enc/trans/transdb.so
7f97a1618000-7f97a1619000 r--p 00002000 08:01 2105270                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/enc/trans/transdb.so
7f97a1619000-7f97a161a000 rw-p 00003000 08:01 2105270                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/enc/trans/transdb.so
7f97a161a000-7f97a161c000 r-xp 00000000 08:01 1977261                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/enc/encdb.so
7f97a161c000-7f97a181b000 ---p 00002000 08:01 1977261                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/enc/encdb.so
7f97a181b000-7f97a181c000 r--p 00001000 08:01 1977261                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/enc/encdb.so
7f97a181c000-7f97a181d000 rw-p 00002000 08:01 1977261                    /home/vagrant/.rbenv/versions/2.3.1/lib/ruby/2.3.0/x86_64-linux/enc/encdb.so
7f97a181d000-7f97a7d44000 r--p 00000000 08:01 787270                     /usr/lib/locale/locale-archive
7f97a7d44000-7f97a7d46000 r-xp 00000000 08:01 656724                     /usr/lib64/libfreebl3.so
7f97a7d46000-7f97a7f45000 ---p 00002000 08:01 656724                     /usr/lib64/libfreebl3.so
7f97a7f45000-7f97a7f46000 r--p 00001000 08:01 656724                     /usr/lib64/libfreebl3.so
7f97a7f46000-7f97a7f47000 rw-p 00002000 08:01 656724                     /usr/lib64/libfreebl3.so
7f97a7f47000-7f97a80fd000 r-xp 00000000 08:01 656766                     /usr/lib64/libc-2.17.so
7f97a80fd000-7f97a82fd000 ---p 001b6000 08:01 656766                     /usr/lib64/libc-2.17.so
7f97a82fd000-7f97a8301000 r--p 001b6000 08:01 656766                     /usr/lib64/libc-2.17.so
7f97a8301000-7f97a8303000 rw-p 001ba000 08:01 656766                     /usr/lib64/libc-2.17.so
7f97a8303000-7f97a8308000 rw-p 00000000 00:00 0 
7f97a8308000-7f97a8409000 r-xp 00000000 08:01 656774                     /usr/lib64/libm-2.17.so
7f97a8409000-7f97a8608000 ---p 00101000 08:01 656774                     /usr/lib64/libm-2.17.so
7f97a8608000-7f97a8609000 r--p 00100000 08:01 656774                     /usr/lib64/libm-2.17.so
7f97a8609000-7f97a860a000 rw-p 00101000 08:01 656774                     /usr/lib64/libm-2.17.so
7f97a860a000-7f97a8612000 r-xp 00000000 08:01 656770                     /usr/lib64/libcrypt-2.17.so
7f97a8612000-7f97a8811000 ---p 00008000 08:01 656770                     /usr/lib64/libcrypt-2.17.so
7f97a8811000-7f97a8812000 r--p 00007000 08:01 656770                     /usr/lib64/libcrypt-2.17.so
7f97a8812000-7f97a8813000 rw-p 00008000 08:01 656770                     /usr/lib64/libcrypt-2.17.so
7f97a8813000-7f97a8841000 rw-p 00000000 00:00 0 
7f97a8841000-7f97a8844000 r-xp 00000000 08:01 656772                     /usr/lib64/libdl-2.17.so
7f97a8844000-7f97a8a43000 ---p 00003000 08:01 656772                     /usr/lib64/libdl-2.17.so
7f97a8a43000-7f97a8a44000 r--p 00002000 08:01 656772                     /usr/lib64/libdl-2.17.so
7f97a8a44000-7f97a8a45000 rw-p 00003000 08:01 656772                     /usr/lib64/libdl-2.17.so
7f97a8a45000-7f97a8a5b000 r-xp 00000000 08:01 656792                     /usr/lib64/libpthread-2.17.so
7f97a8a5b000-7f97a8c5b000 ---p 00016000 08:01 656792                     /usr/lib64/libpthread-2.17.so
7f97a8c5b000-7f97a8c5c000 r--p 00016000 08:01 656792                     /usr/lib64/libpthread-2.17.so
7f97a8c5c000-7f97a8c5d000 rw-p 00017000 08:01 656792                     /usr/lib64/libpthread-2.17.so
7f97a8c5d000-7f97a8c61000 rw-p 00000000 00:00 0 
7f97a8c61000-7f97a8c82000 r-xp 00000000 08:01 656759                     /usr/lib64/ld-2.17.so
7f97a8c90000-7f97a8c9a000 r--s 00000000 08:01 668540                     /usr/lib64/libpib-rdmav2.so
7f97a8c9a000-7f97a8cad000 r--s 00000000 08:01 667153                     /usr/lib64/libibverbs.so.1.0.0
7f97a8cad000-7f97a8d48000 r--s 00000000 08:01 669709                     /usr/lib/debug/usr/lib64/libpthread-2.17.so.debug
7f97a8d48000-7f97a8d6b000 r--s 00000000 08:01 656792                     /usr/lib64/libpthread-2.17.so
7f97a8d6b000-7f97a8d72000 r--s 00000000 08:01 921249                     /usr/lib64/gconv/gconv-modules.cache
7f97a8d72000-7f97a8e78000 rw-p 00000000 00:00 0 
7f97a8e7b000-7f97a8e7d000 rw-p 00000000 00:00 0 
7f97a8e7d000-7f97a8e7e000 ---p 00000000 00:00 0 
7f97a8e7e000-7f97a8e82000 rw-p 00000000 00:00 0                          [stack:13141]
7f97a8e82000-7f97a8e83000 r--p 00021000 08:01 656759                     /usr/lib64/ld-2.17.so
7f97a8e83000-7f97a8e84000 rw-p 00022000 08:01 656759                     /usr/lib64/ld-2.17.so
7f97a8e84000-7f97a8e85000 rw-p 00000000 00:00 0 
7f97a8e85000-7f97a916f000 r-xp 00000000 08:01 1976767                    /home/vagrant/.rbenv/versions/2.3.1/bin/ruby
7f97a936f000-7f97a9374000 r--p 002ea000 08:01 1976767                    /home/vagrant/.rbenv/versions/2.3.1/bin/ruby
7f97a9374000-7f97a9375000 rw-p 002ef000 08:01 1976767                    /home/vagrant/.rbenv/versions/2.3.1/bin/ruby
7f97a9375000-7f97a9387000 rw-p 00000000 00:00 0 
7f97aa057000-7f97aa4b7000 rw-p 00000000 00:00 0                          [heap]
7ffd88ff0000-7ffd897ef000 rw-p 00000000 00:00 0 
7ffd897ef000-7ffd897f1000 r-xp 00000000 00:00 0                          [vdso]
ffffffffff600000-ffffffffff601000 r-xp 00000000 00:00 0                  [vsyscall]


[NOTE]
You may have encountered a bug in the Ruby interpreter or extension libraries.
Bug reports are welcome.
For details: http://www.ruby-lang.org/bugreport.html

Aborted
```

**Comment by hiroyuki-sato** [comment]
``` C
 707 int ibv_cmd_create_qp(struct ibv_pd *pd,
 708                       struct ibv_qp *qp, struct ibv_qp_init_attr *attr,
 709                       struct ibv_create_qp *cmd, size_t cmd_size,
 710                       struct ibv_create_qp_resp *resp, size_t resp_size)
 711 {
 712         IBV_INIT_CMD_RESP(cmd, cmd_size, CREATE_QP, resp, resp_size);
 713 
 714         cmd->user_handle     = (uintptr_t) qp;
 715         cmd->pd_handle       = pd->handle;
 716         cmd->send_cq_handle  = attr->send_cq->handle;
 717         cmd->recv_cq_handle  = attr->recv_cq->handle;
 718         cmd->srq_handle      = attr->srq ? attr->srq->handle : 0;
 719         cmd->max_send_wr     = attr->cap.max_send_wr;
```

**Comment by hiroyuki-sato** [comment]
Fix: 4b6687516aff97e279696dc54ab7dcee13e11497
